Resources/Locale/en-US/_NF/accent/goblin_accent.ftl

```diff
@@ -727,11 +777,61 @@ accent-goblin-words-replace-355 = touted
 accent-goblin-words-replace-356 = touts
 accent-goblin-words-replace-357 = toutin’
 accent-goblin-words-replace-358 = wo’ah
-accent-goblin-words-replace-359 = wever
+accent-goblin-words-replace-359 = wevah
 accent-goblin-words-replace-360 = wiv
 accent-goblin-words-replace-361 = wivout
 accent-goblin-words-replace-362 = screeve
 accent-goblin-words-replace-363 = screeves
 accent-goblin-words-replace-364 = screevin’
 accent-goblin-words-replace-365 = screeved
 accent-goblin-words-replace-366 = screeved
+accent-goblin-words-replace-367 = dis
+accent-goblin-words-replace-368 = dat
+accent-goblin-words-replace-369 = den
+accent-goblin-words-replace-370 = 'em
+accent-goblin-words-replace-371 = dey
+accent-goblin-words-replace-372 = dese
+accent-goblin-words-replace-373 = ya
+accent-goblin-words-replace-374 = dere
+accent-goblin-words-replace-375 = course
+accent-goblin-words-replace-376 = uvah
+accent-goblin-words-replace-377 = uvas
+accent-goblin-words-replace-378 = uverwise
```

Comment:
Would recommend double Vs here to avoid reading as a long U.  Used these a bit moiself, oi 'ave.
```suggestion
accent-goblin-words-replace-376 = uvvah
accent-goblin-words-replace-377 = uvvas
accent-goblin-words-replace-378 = uvvawise
```

Reply:
Yeah, replaced those as you suggested while was adding more words to the table